velox/core/PlanNode.h

```diff
@@ -246,6 +249,50 @@ class ValuesNode : public PlanNode {
   const bool parallelizable_;
 };
 
+class ArrowStreamNode : public PlanNode {
+ public:
+  ArrowStreamNode(
+      const PlanNodeId& id,
+      const RowTypePtr& outputType,
+      std::shared_ptr<ArrowArrayStream> arrowStream,
+      memory::MemoryPool* pool)
```

Comment:
We should not pass memory pool to the plan node. The operator can get access to memory pool via Operator::pool() getter.

Reply:
Fixed.